scripts/postprocessing.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import pandas as pd
+from metriq import MetriqClient
+from metriq.models.result import ResultCreateRequest
+
+METRIQ_TOKEN = os.getenv("METRIQ_TOKEN")
+RESULTS_PATH = os.path.abspath(os.path.join( os.path.dirname( __file__ ),"..", "benchmarking", "results"))
+TASKS = {"25": "ex1_226.qasm", "26": "ex1_226.qasm (Aspen)", "27": "ex1_226.qasm (Rochester)"}
+
+def submit_all(task):
+  filenames = os.listdir(RESULTS_PATH)
+
+  for filename in filenames:
+    architectures = ["aspen", "rochester"]
+    for arch in architectures:
+      if arch in filename and arch in task.lower():
+        file_path = os.path.join(RESULTS_PATH, f"{filename}")
+        df = pd.read_csv(file_path, sep='|')
+        process_results(df, task)
+
+def process_results(dataframe, task):
+  client = MetriqClient(token=METRIQ_TOKEN)
+  # print(client.hello())
+
+  metrics = ["Circuit depth", "Gate count"]
+  for metric in metrics:
+    result = ResultCreateRequest()
+    result.task = task
+    result.platform = dataframe["Platform"].iloc[0]
+    result.method = "Qiskit compilation"
+    result.metricName = metric
+    result.metricValue = dataframe[metric].mean()
+    result.evaluatedAt = dataframe["Date"].iloc[0]
+    result.isHigherBetter = "false"
+    # result.sampleSize = len(dataframe.index) object has no field "sampleSize"
+
+    # Get extra info and add to notes
+    sample_size = len(dataframe.index)
+    metric_std = dataframe[metric].std()
+    opt_level = dataframe["Opt level"].iloc[0]
+    version = dataframe["Method"].iloc[0].split(" ")[1]
+    result.notes = f"Stdev: {round(metric_std,3)}, Optimization level:{opt_level}, qiskit-terra version:{version}, sample size: {sample_size}"
+
+  # TODO:
+  # - add result item client
+  # - create metriq submission item
+  # - submit item though metriq API
+
+  # client.result_add(result) #missing 1 required positional argument: 'submission_id'
```

Comment:
@vprusso @WrathfulSpatula I'm hitting an error when adding a result item to my client instance:
`TypeError: result_add() missing 1 required positional argument: 'submission_id'`
In my understanding, and following the examples in metriq-client, I should be able to add my result items independent of a submission.
Or do I need to add a client submission first, then add my results to it afterwards?
Just trying to figure out the expected order of things. `submission_add()` makes a post request, but ideally I would only use it when I have all my result items ready to go, if that makes sense.

Reply:
Hi @karlaspuldaro. 

Thanks for the question. Yes, indeed. Result objects are keyed to submissions, so it wouldn't be possible to add a result in isolation (it must be associated with a submission, and hence, would require an associated `submission_id` to be keyed to).

Does that workflow work for you and make sense? Happy to help or clarify if not.

Reply:
Thank you @vprusso 
I'll create a submission item for these results. 
On another note, is there a way to pick what kind of chart the results are displayed in metriq? As we calculate the average and standard dev of our experiment runs, it would be beneficial to show the distribution somehow, ideally with a box chart.

Reply:
@karlaspuldaro Currently, we only offer the single type of general chart and its associated options. However, the Metriq team has talked about implementing other options to visualize data, and I could ask them if we could implement a special or general chart option for you. (I'm willing to implement it, but we do have other development priorities, too.)

Reply:
@WrathfulSpatula That would be amazing, I could also help on the implementation if you could provide some guidance :) Is there an open issue for expanding the charts?

Reply:
@karlaspuldaro I just created this: https://github.com/unitaryfund/metriq-app/issues/862

There's no user story yet, but feel free to add any specifics about the requested design that you think are helpful to understand your need as a user, and I'll speak with the team.

Reply:
@WrathfulSpatula thx so much! Will do :)

Reply:
@karlaspuldaro I just saw some examples of your box plots: I suspect that the "standard error" options on Metriq results can be updated (on our end with a bit of additional development) to generate box plots for your data. Also, speaking with the rest of the team, we thought we could go ahead with this.

It happens to be that we're in the process of a redesign of the task plots view in the web app, and I'll add this a work item (or user story) to that "sprint."